### DISCUSSION TITLE
Testcontainers wrapper

### DISCUSSION THREAD

**Discussion by ivanyu** [fsouza/fake-gcs-server#1339] [Show and tell]
Hello!

I hope this would be interesting to you. We made a Testcontainers Java wrapper for `fake-gcs-server`: https://github.com/Aiven-Open/testcontainers-fake-gcs-server

**Comment by fsouza** [comment]
Nice, thanks for sharing! Do you want to send a PR adding a link to the README?

**Comment by ivanyu** [comment]
Sure! What section would you think be good for this?
